odl/discr/diff_ops.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, space, axis=0, method='forward', padding_method=None,
             accuracy in the interior.
         """
         if not isinstance(space, DiscreteLp):
-            raise TypeError('space {!r} is not a DiscreteLp instance.'
+            raise TypeError('`space` {!r} is not a DiscreteLp instance.'
```

Comment:
General question: do we want back-ticks in error messages or not? Consistency needed.